### PR TITLE
Test Node.isTiedToVcc()

### PR DESCRIPTION
**Pull request by eddieh-xlnx**

### CODE REVIEW

File: test/src/com/xilinx/rapidwright/device/TestNode.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class TestNode {
+    @ParameterizedTest
+    @CsvSource({
+            "xcvu440-flga2892,LAGUNA_TILE_X161Y659/VCC_WIRE",
+            "xcvu440-flga2892,LAGUNA_TILE_X161Y659/VCC_WIRE0",
+    })
```
Comment (eddieh-xlnx) [inline test/src/com/xilinx/rapidwright/device/TestNode.java:34]:
```suggestion
    @CsvSource({
             "xcvu440-flga2892,LAGUNA_TILE_X161Y659/VCC_WIRE",
             "xcvu440-flga2892,LAGUNA_TILE_X161Y659/VCC_WIRE0",
    })
```

**Comment by clavin-xlnx** [comment]
Although I can recognize the confusion, it looks like RapidWright is faithfully reporting what Vivado is reporting regarding VCC connectivity:

```
Vivado% link_design -part xcvu440-flga2892-2-e
#...
Vivado% report_property [get_nodes LAGUNA_TILE_X161Y659/VCC_WIRE]
Property           Type    Read-only  Visible  Value
#...
IS_BAD             bool    true       true     0
IS_COMPLETE        bool    true       true     1
IS_GND             bool    true       true     0
IS_INPUT_PIN       bool    true       true     0
IS_OUTPUT_PIN      bool    true       true     0
IS_PIN             bool    true       true     0
IS_VCC             bool    true       true     0
#... 
Vivado% report_property [get_nodes LAGUNA_TILE_X161Y659/VCC_WIRE0]
Property           Type    Read-only  Visible  Value
#...
IS_BAD             bool    true       true     0
IS_COMPLETE        bool    true       true     1
IS_GND             bool    true       true     0
IS_INPUT_PIN       bool    true       true     0
IS_OUTPUT_PIN      bool    true       true     0
IS_PIN             bool    true       true     0
IS_VCC             bool    true       true     1
#...
```

Should we close this PR or deviate from Vivado?

**Comment by clavin-xlnx** [comment]
image

Interestingly, the PIP Junction `IS_VCC` is set to 1, but strangely the node is not:
image

**Comment by eddieh-xlnx** [comment]
Ah, that's frustrating. I suppose RapidWright should only be reporting what Vivado gives up rather than embellishing its own meanings. Looks like I can't rely on this property then!

**Pull request closed without merging**